mappings/net/minecraft/client/render/Camera.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS ctf net/minecraft/client/render/Camera
+	FIELD a ready Z
+	FIELD b area Lbfi;
+	FIELD c focussedEntity Lahw;
+	FIELD d pos Lcqg;
+	FIELD e blockPos Leu$a;
+	FIELD i pitch F
+	FIELD j yaw F
+	FIELD k thirdPerson Z
+	FIELD l inverseView Z
+	METHOD a updateEyeHeight ()V
+	METHOD a (DDD)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+	METHOD a addRotation (FF)V
+		ARG 1 yaw
+		ARG 2 pitch
+	METHOD a update (Lbfi;Lahw;ZZF)V
+		ARG 1 area
+		ARG 2 focussedEntity
+		ARG 3 thirdPerson
+		ARG 4 inverseView
+		ARG 5 partialTicks
```

Comment:
`partialTicks` is Voldemap name; I believe `deltaTicks` is Fabric standard, but I could be wrong.

Reply:
Ah I thought it was just the generic word for a percent between updates

Reply:
My bad, looks like the standard is `tickDelta`